src/components/EnhancedMap/LayerToggle/LayerToggle.js

```diff
@@ -29,85 +29,74 @@ export class LayerToggle extends Component {
   render() {
     const baseSources = _filter(this.props.layerSources, source => !source.overlay)
 
-    const layerButtons = _map(baseSources, layer => (
-      // eslint-disable-next-line jsx-a11y/anchor-is-valid
-      <a className={classNames('dropdown-item',
-                               {'is-active': this.props.source.id === layer.id})}
-         key={layer.id}
+    const layerListItems = _map(baseSources, layer => (
+      <li key={layer.id}>
+        <button className={classNames('test', this.props.source.id === layer.id ? 'mr-text-green-lighter' : 'mr-text-current')}
```

Comment:
Looks like maybe a 'test' class got left in? if that's not needed, you can also get rid of the classNames() and just pass the ternary into className={}